modules/programs/obs-studio.nix

```diff
@@ -39,7 +48,7 @@ in {
   };
 
   config = mkIf cfg.enable {
-    home.packages = [ package ];
```

Comment:
Can you get rid of the package variable in favor of cfg.package then?

Reply:
done